Api/Src/Controllers/GradeSheetController.ts

```diff
@@ -0,0 +1,83 @@
+import GradeSheetModel from '../Models/GradeSheet';
+import GradeSheetService from '../Services/GradeSheetService';
+import { Request, Response } from 'express';
+import * as mongoose from 'mongoose';
+
+export default class UserController {
```

Comment:
It's a default export, so doesn't cause any problems, but should be GradeSheetController

Reply:
Corrected 👍